rootless-install.sh

```diff
@@ -101,8 +101,8 @@ checks() {
 			>&2 echo "- or simply log back in as the desired unprivileged user (ssh works for remote machines)"
 			exit 1
 		fi
-		export XDG_RUNTIME_DIR="/tmp/docker-$(id -u)"
-		mkdir -p "$XDG_RUNTIME_DIR"
+		export XDG_RUNTIME_DIR="$HOME/.docker/xrd"
```

Comment:
As discussed on Slack (for a follow-up) I think we should find a way where not require `XDG_RUNTIME_DIR` to be set. This env-var is not "owned" by docker, so we should not set it. Instead, we could use a default path for the non-`XDG_RUNTIME_DIR` case (perhaps clean up the directory on daemon start)

Reply:
Do we want that feature in 20.0X? or in 20.1X? (21.0X?)

Reply:
If we have it ready before 20.0x, I think that'd be good to have